packages/appstore/src/modules/tour-guide/tour-guide.tsx

```diff
@@ -94,7 +89,7 @@ const TourGuide = () => {
         );
     }
 
-    const high_risk_and_eu_check = content_flag === ContentFlag.HIGH_RISK_CR || content_flag === eu_user;
+    const low_risk = content_flag === ContentFlag.LOW_RISK_CR_NON_EU;
```

Comment:
The regulation part of tour guide should be visible both for` .LOW_RISK_CR_NON_EU` an `LOW_RISK_CR_EU`